### PR TITLE
Remove duplicated includes in gui/graf2d/graf3d/geom/proof classes

### PR DESCRIPTION
**Pull request by linev**
Just remove line where same include appears second time

### CODE REVIEW

**Comment by linev** [comment]
@bellenot or @couet Can you approve PR?

File: net/http/src/THttpServer.cxx [hunk 1]
```diff
@@ -32,8 +32,7 @@
 
 #include <string>
 #include <cstdlib>
-#include <stdlib.h>
-#include <string.h>
+#include <cstring>
```
Comment (Axel-Naumann) [inline net/http/src/THttpServer.cxx:35]: Might need prefix of `std::` for `strstr` etc.

Comment (linev) [reply net/http/src/THttpServer.cxx:35]: It is compiled on all platforms. Once it is really appears - I will fix it.

**Comment by linev** [comment]
It is trivial change - merging it

**Pull request merged**